packages/manager/apps/dedicated/client/app/account/user/infos/user-infos.service.js

```diff
@@ -57,6 +57,27 @@ export default class UserAccountInfosService {
       });
   }
 
+  fetchMarketingConsentDecision() {
+    return this.$http.get('/me/marketing').then((response) => {
+      if (response.status < 300) {
+        return response.data;
+      }
+      return this.$q.reject(response);
+    });
```

Comment:
**question:** If the status is `>300`, why is it resolved? It should go to `.catch()` no?

Reply:
It resolves for any 200 status and reject it for 300+ status